### PR TITLE
fix for NUTCH-2027

### PR DESCRIPTION
**Pull request by asitang**

### CODE REVIEW

**Comment by chrismattmann** [comment]
@asitang did you forget to attach SeedResource?

File: src/java/org/apache/nutch/service/resources/SeedResource.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
+
+
+/*******************************************************************************
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/service/resources/SeedResource.java]: please don't invent new ways to show the header. Just use the way the other files have a header. Please see them.

File: src/java/org/apache/nutch/service/model/request/SeedList.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package org.apache.nutch.service.model.request;
+
+import java.util.ArrayList;
+
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/service/model/request/SeedList.java]: class javadoc, please.

File: src/java/org/apache/nutch/service/resources/SeedResource.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+import javax.ws.rs.core.Response.Status;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.nutch.service.model.request.SeedList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+
+
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/service/resources/SeedResource.java]: class javadoc please.

**Comment by sujen1412** [comment]
Hi @asitang, just saw the code and I think the you will need to format the code according to http://svn.apache.org/viewvc/nutch/branches/2.x/eclipse-codeformat.xml

**Pull request closed without merging**